backend/compact-connect/lambdas/provider-data-v1/data_model/schema/privilege.py

```diff
@@ -20,6 +20,7 @@ class PrivilegeRecordSchema(BaseRecordSchema):
     status = String(required=True, allow_none=False, validate=OneOf(['active', 'inactive']))
     dateOfIssuance = Date(required=True, allow_none=False)
     dateOfExpiration = Date(required=True, allow_none=False)
+    compactTransactionId = String(required=False, allow_none=False)
```

Comment:
What's the implication of having an optional field, where "None" is not allowed, and is that we intended?

Reply:
It means you don't have to include the key in the object, but if you do, it must have a string value (it's not nullable)

Reply:
and yes, according to @jusdino there are cases where a licensee will skip purchasing privileges in the case that they move home states, so we want to leave this as optional.